riotfile.py

```diff
@@ -48,7 +48,7 @@
             name="mypy",
             command="mypy --install-types --non-interactive {cmdargs}",
             pkgs={
-                "mypy": latest,
+                "git+https://github.com/p403n1x87/mypy": "@fix-handle-prefix",
```

Comment:
this should be done separately?

Reply:
I'm not sure we'd really want to do this anyway. I've put it so that we can see the `mypy` check pass in CI. But if we think this is OK we should do it in a separate PR. At least one other change would have to be pulled out out here as it's not quite related, so a bit of clean-up is still needed.